docs/calendar-review.md

```diff
@@ -0,0 +1,212 @@
+
+### Alternative 1: No Calendar in the Data Model
+
+An alternative approach to the current Temporal design would have been to store calendar information separately from Temporal instances. This approach would have the advantage of avoiding the Unexpected Calendar Problem, but at significant ergonomic cost:
+
+*   Calendar-specific fields (e.g. `month`, `day`, `year`) and convenience properties (e.g. `daysInMonth`, `inLeapYear`) could not be parameterless getters anymore. Instead, users would have to call methods to get calendar-specific values, e.g. `date.calendarMonth('chinese')`
+*   Many methods (`add`, `subtract`, `round`, `until`, `since`, `with`, maybe `toPlainMonthDay`) would require a calendar option to be able to use calendar-specific months, days, or years. Example: \
+`date.add({months: 2}, {calendar: 'chinese'})`
+*   Chaining multiple operations (e.g. `.add({months: 2}).with({day: 1})`) gets less ergonomic and more bug-prone because the same calendar value must be used in all chained calls. 
+*   The `Temporal.PlainMonthDay` type would become challenging to reason about without a calendar attached because month/day values like birthdays and holidays are inherently calendar-specific and have no ISO counterpart. This is especially true in lunisolar calendars like Hebrew and Chinese where ordinal month numbers vary from year to year for the same month. (Current Temporal uses a string <code>[monthCode](https://tc39.es/proposal-temporal/docs/plaindate.html#monthCode)</code> field to describe year-independent months.) For example, a birthday on 12 Adar I in the Hebrew calendar is currently modelled as \
+<code>  Temporal.PlainMonthDay.from( \
+    { monthCode: '05L', day: 12, calendar: 'hebrew' } \
+  ); \
+</code>It’s not clear how we’d model holidays if the calendar isn’t stored in the instance.
+*   Code that wants to return a date along with its calendar can no longer return a single Temporal instance. Instead, it must return a compound object or serialized string. For example, a date picker that returned a date in the user’s preferred calendar could no longer return a <code>Temporal.PlainDate</code>. Instead, it’d have to return a tuple or compound object like <code>{date, calendar}</code>.  Passing around compound objects (or extra calendar parameters) would complicate all calendar-using Temporal code.
+
+The Temporal champions believe that the ergonomic challenges of this alternative are worse than the work required to mitigate the Unexpected Calendar Problem. 
+
+### Alternative 2: Separate ISO and Calendar Properties
+
+It would be possible to retain the calendar in Temporal objects’ data model without being vulnerable to the Unexpected Calendar Problem, but doing so would involve a significant increase in Temporal surface area. Below is one way it could be done. 
```

Comment:
```suggestion
It would be possible to retain the calendar in Temporal objects' data model without being vulnerable to the Unexpected Calendar Problem, but doing so would involve a significant increase in Temporal surface area. Below is one way it could be done. 
```